src/App.jsx

```diff
@@ -1,41 +1,85 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
+import { getPosts, getUserPosts } from './api/posts';
+import data from './api/users.json';
```

Comment:
Import is not named (`{ name }`), so you can control the name of the variable.
```suggestion
import users from './api/users.json';
```